### ISSUE TITLE
Settings ignored if set after import

### ISSUE THREAD

**Issue by koleror** [realpython/django-slow-tests#24]
I need to add some custom features to `DiscoverSlowestTestsRunner`, so to do so, I di the following:

```python
from django_slowtests.testrunner import DiscoverSlowestTestsRunner

class CustomDiscoverRunner(DiscoverSlowestTestsRunner):
    pass

TEST_RUNNER = 'myapp.settings.CustomDiscoverRunner'
NUM_SLOW_TESTS = 20
SLOW_TEST_THRESHOLD_MS = 200
```

The problem with that is that, as you're loading the settings at the module level (here : https://github.com/realpython/django-slow-tests/blob/master/django_slowtests/testrunner.py#L21-L22), my settings are ignored and the default values used.

To fix that, you simply need to do the getattr at runtime instead of module level.
I could also define `NUM_SLOW_TESTS` and  `SLOW_TEST_THRESHOLD_MS` before the import, but that would make my linter unhappy and it might easily be broken in the future.
I can make a PR for that if you want.
What do you think?

**Comment by mjhea0** [comment]
Thanks for all the PRs. Right now they are all breaking due to Flake8. Can you address what it's complaining about? Thanks again!

**Comment by koleror** [comment]
Only one of the PRs is breaking because of flake8. I commented it. 
But they are all breaking on 3 flavors and I don't think it's related to my code, is it?

**Comment by mjhea0** [comment]
I don't think so. Do you want to become the maintainer of this package per chance?

**Comment by koleror** [comment]
Why not :)

**Comment by mjhea0** [comment]
@dbader any issues with this?

**Comment by dbader** [comment]
> @dbader any issues with this?

Nope, let's do it :-) Adding you as a contributor with write access now @koleror 

I'm guessing this is published to PyPI under your account @mjhea0?

**Comment by mjhea0** [comment]
Yes. @koleror - what's your PyPI username?

**Comment by koleror** [comment]
@mjhea0 https://pypi.org/user/koleror/

**Comment by mjhea0** [comment]
"User 'koleror' does not have a verified primary email address and cannot be added as a Maintainer for project. "

**Comment by koleror** [comment]
Fixed

**Comment by mjhea0** [comment]
You have been added.

**Comment by koleror** [comment]
Thx. So should I use my fork as the new project's official repo or get write on this one?

**Comment by dbader** [comment]
> Thx. So should I use my fork as the new project's official repo or get write on this one?

You have write access here now, so why don't we keep using this repo :)

**Comment by koleror** [comment]
Nope, I don't think I do: 
```
git push --set-upstream origin test [enter/↑/↓/ctrl+c]
ERROR: Permission to realpython/django-slow-tests.git denied to koleror.
fatal: Could not read from remote repository.```

**Comment by dbader** [comment]
@koleror https://github.com/realpython/django-slow-tests/invitations

**Comment by koleror** [comment]
Hmm did not get a notif for that. Thx
